### DISCUSSION TITLE
What happened to Upscalers?

### DISCUSSION THREAD

**Discussion by Art3mis-Ghub** [vladmandic/sdnext#2313] [General]
I've been away for three months, had to reinstall SD.Next and now I can't see any tabs for scaling images anymore? What's up with that? I've installed NMKD to the ESRGAN folder like I used to, yet nothing.

**Comment by vladmandic** [comment]
what do you mean? they're still there.

**Comment by Tillerz** [comment]
Is it about "img2img" resize? 

<img width="816" alt="57549fd0c2a47794571202561aa7e5d4" src="https://github.com/vladmandic/automatic/assets/63370525/48eff598-8b81-4efe-90bb-05ed5d0ab62d">

Because I do not see any rescaler selection there either. But I also cannot remember if it ever had any. I always directly rescale in txt2img.

"Process image" has a rescaler dropdown:

<img width="693" alt="72e0823b8bc58e35f863d0699ddcbef1" src="https://github.com/vladmandic/automatic/assets/63370525/536e97f1-266e-4ba4-afa9-4452eb237b36">

And here it is in "txt2img":

<img width="410" alt="9daee315d03766dd83584e1d9c04ac40" src="https://github.com/vladmandic/automatic/assets/63370525/68a07245-81c1-407a-9513-5e23ba3963e1">

**Reply by vladmandic** [reply]
img2img never had its own upscaler selection, it uses whatever is set in settings -> postprocessing -> default upscaler for image resize operations

or you can enable postprocessing upscalers globally in settings -> postprocessing -> enable additional postprocessing operations

**Reply by Tillerz** [reply]
Then there is no upscaler missing, each page has it, directly or borrowing from another one. :)
